src/cdk/datetime/datepicker.md

```diff
@@ -0,0 +1,73 @@
+The datepicker CDK provides a foundation upon which more concrete Material datepicker 
+implementations can be built. The datepicker CDK will be a generic datepicker with all of the
+necessary basic datepicker functionality needed for the definition of a datepicker. Users will be
+able to extend this datepicker CDK with all of the basic datepicker functionality as well as create
+their own views of the calendar. In this way, the users will be able to customized the datepicker
+to allow for more flexibility when utilizing the datepicker component in applications.
```

Comment:
I generally like to open docs for the cdk with a single sentence summary of what the package contains, and then go into more specific categories from there. So for this, I would do something like

```markdown
The `datetime` package offers building blocks for custom date and time-based components.

### DateAdapter
The `DataAdapter` provides a uniform way to deal with different date representations. ...


### CdkDatepicker Component
The `CdkDatepicker` component provides...


#### Building a custom datepicker on top of CdkDatepicker
...
```

Reply:
I created documentation for the `DateAdapter` in `date-adapter.md`. Should I merge this into one?

Reply:
Yeah, I think you also need to rename this to `datetime.md` I'm pretty sure our docs site just looks for a `.md` file with the same name as the directory

Reply:
Renamed!